docs/man_pages/project/configuration/generate-icon.md

```diff
@@ -0,0 +1,34 @@
+generate icon
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder generate icon <Path>` [--force]
+
+Generates icons from a single high-resolution image and saves them to the current project
```

Comment:
Generates icons for all mobile platforms from a single high-resolution image and saves them to the `App_Resources` folder of the current project.